src/domainUser/domainUser.interface.ts

```diff
@@ -6,5 +6,6 @@ export interface IDomainUser {
   domain: string;
   name: string;
   fullString?: string;
+  UID?: string;
```

Comment:
adfsUID
adfsUid
adfsUId

Reply:
Done and chenge fullstring field to uniqueID